data/projects/semesters/fa19/index.js

```diff
@@ -1,7 +1,7 @@
 import CCLData from "./ccl";
-import LAHData from "./lah";
+import LAHData from "../sp20/lah";
 import MLIDATA from "./mli";
-import YMCADATA from "./ymca";
+import YMCADATA from "../sp20/ymca.js";
```

Comment:
nitpick but we can probably remove the `.js` to be consistent with import naming